dashboard/src/components/shared/format/Money.vue

```diff
@@ -25,6 +35,33 @@ export default class Money extends Vue {
     return this.chainProperties.tokenSymbol
   }
 
-}
+  public created() {
```

Comment:
use `mounted` instead of `created`.

Reply:
changed but I don't know why do you want to use `mounted` here
with `created` hook we can get data faster

Reply:
I always used mounted for fetching. However, it looks like it doesn't matter.
https://stackoverflow.com/a/57089636

Reply:
I always use `created` for fetching because it's executed before mounted and I don't need DOM elements here